### ISSUE TITLE
Function Error: 'AggregatorMaxFunction' eval unimplemented

### ISSUE THREAD

**Issue by BohuTANG** [datafuselabs/datafuse#538]
**Describe the bug**
client:
```
mysql> SELECT number%2 as c1, number%3 as c2, MAX(number) FROM numbers(10000) GROUP BY c1, c2 HAVING MAX(number)>9996;
ERROR 1105 (HY000): Code: 2, displayText = Function Error: 'AggregatorMaxFunction' eval unimplemented.
```

server:
```
[2021-05-14T01:25:07Z ERROR fuse_query::servers::mysql::endpoints::endpoint_on_query] OnQuery Error: Code: 2, displayText = "Function Error: \'AggregatorMaxFunction\' eval unimplemented".
    
       0: common_exception::exception::ErrorCodes::UnImplement
                 at common/exception/src/exception.rs:38:41
          common_functions::function::IFunction::eval
                 at common/functions/src/function.rs:24:21
       1: <common_functions::comparisons::comparison::ComparisonFunction as common_functions::function::IFunction>::eval
                 at common/functions/src/comparisons/comparison.rs:83:21
       2: <fuse_query::pipelines::transforms::transform_filter::FilterTransform as fuse_query::pipelines::processors::processor::IProcessor>::execute::{{closure}}::{{closure}}
                 at fusequery/query/src/pipelines/transforms/transform_filter.rs:74:32
          <fuse_query::pipelines::transforms::transform_filter::FilterTransform as fuse_query::pipelines::processors::processor::IProcessor>::execute::{{closure}}::{{closure}}
                 at fusequery/query/src/pipelines/transforms/transform_filter.rs:85:46
          <tokio_stream::stream_ext::filter_map::FilterMap<St,F> as futures_core::stream::Stream>::poll_next
                 at /home/bohu/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-stream-0.1.5/src/stream_ext/filter_map.rs:46:38
       3: <core::pin::Pin<P> as futures_core::stream::Stream>::poll_next
                 at /home/bohu/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/futures-core-0.3.14/src/stream.rs:129:9
          <tokio_stream::stream_ext::collect::Collect<T,U> as core::future::future::Future>::poll
                 at /home/bohu/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-stream-0.1.5/src/stream_ext/collect.rs:71:37
       4: <F as futures_core::future::TryFuture>::try_poll

```

**Comment by BohuTANG** [comment]
It seems that here need an expression action for having

**Comment by sundy-li** [comment]
It has been resolved after #550 
```
mysql> SELECT number%2 as c1, number%3 as c2, MAX(number) FROM numbers(10000) GROUP BY c1, c2 HAVING MAX(number)>9996;
+------+------+-------------+
| c1   | c2   | MAX(number) |
+------+------+-------------+
|    1 |    0 |        9999 |
|    0 |    2 |        9998 |
|    1 |    1 |        9997 |
+------+------+-------------+
3 rows in set (0.02 sec)
```
